packages/p2p/src/components/composite-calendar/composite-calendar-mobile/composite-calendar-mobile.tsx

```diff
@@ -0,0 +1,197 @@
+import React from 'react';
+import { Button, DatePicker, InputField, MobileDialog } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
```

Comment:
shouldn't we use the Localize from P2P package here since this component is inside P2P?

Reply:
nice catch

Reply:
oops.. i missed it. have updated..